### DISCUSSION TITLE
Documentation : list UI Component options as a subheader

### DISCUSSION THREAD

**Discussion by foochris** [buefy/buefy#3993] [Ideas]
### Description
As a new user, it may be helpful to see the options available for each UI Component. In `Steps`, for example, a list could be displayed below the subheading, 'Response horizontal process steps', that displays Dynamic, Icons, Sizes, Types and API.

Currently, a user has to scroll the entire page to see all available subheadings / subtopics.

### Why Buefy need this feature
Having the list of options would allow a new user to efficiently determine if the component is suitable to their requirement(s).

**Comment by jtommy** [comment]
Nice idea

**Comment by foochris** [comment]
@jtommy The implementation could be similar to the right hand column in Bootstrap Vue:

https://bootstrap-vue.js.org/docs/components/table

**Comment by jonagoldman** [comment]
I also like the aproach used in [IBM Components](https://www.carbondesignsystem.com/components/data-table/usage).

**Sections Quick Jump**
Main sections as tabs. Smaller sections as links.
![](https://i.ibb.co/4TxPMGF/1.png)

**Live Demo Options**
Theme and variants dropdowns on top, component options to the right.
![](https://i.ibb.co/Kq9sdzZ/2.png)

**Comment by wesdevpro** [comment]
Hello, from three years in the future 🤣 Does anyone know if I should just close this issues or should this still be a goal for Buefy v2?
@jtommy @kikuomax @wesdevpro
